pkg/utils/grub.go

```diff
@@ -188,7 +192,14 @@ func (g Grub) Install(target, rootDir, bootDir, grubConf, tty string, efi bool,
 		// providing a generic package
 
 		// Shim is not available in Alpine + rpi
-		model, err := utils.OSRelease("KAIROS_MODEL", filepath.Join(cnst.ActiveDir, "etc/os-release"))
+		var model string
+		model, err = utils.OSRelease("KAIROS_MODEL", filepath.Join(cnst.ActiveDir, "etc/kairos-release"))
+		if err != nil {
+			model, err = utils.OSRelease("KAIROS_MODEL", filepath.Join(cnst.ActiveDir, "etc/os-release"))
```

Comment:
maybe worth a comment that we keep this for historical reasons

Reply:
only line I forgot to add the fallback comment lol